### ISSUE TITLE
aws_cognito_identity_pool should export arn

### ISSUE THREAD

**Issue by youngbob** [terraform-providers/terraform-provider-aws#4703]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Cognito Identity Pools, like most AWS resources, have an ARN in addition to their ID that is used by other AWS services as a reference. The aws_cognito_identity_pool resource only exports an ID.

### New or Affected Resource(s)

* aws_cognito_identity_pool

### Potential Terraform Configuration

```hcl
resource "aws_cognito_identity_pool" "main" {
  identity_pool_name               = "identity pool"
}

resource "aws_iam_role_policy" "cognito_sync" {
  name = "cognito_sync"

  policy = <<EOF
{
  "Version": "2012-10-17",
  "Statement":[{
      "Effect":"Allow",
      "Action":"cognito-sync:*",
      "Resource":["${aws_cognito_identity_pool.main.arn}/identity/$${cognito-identity.amazonaws.com:sub}/*"]
      }]
  }
EOF
}
```

### References

* https://docs.aws.amazon.com/cognito/latest/developerguide/iam-roles.html

**Comment by bflad** [comment]
The new `arn` attribute will be released with v1.22.0 of the AWS provider, middle of this week.

**Comment by bflad** [comment]
This has been released in [version 1.22.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1220-june-05-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
